deployments/ansible/molecule/custom_vars/custom_collector_config.yml

```diff
@@ -24,7 +24,7 @@ processors:
 
 exporters:
   logging/debug:
-    loglevel: debug
+    verbosity: detailed
```

Comment:
Let's revert this since this file is only used for testing installation of an older version of the collector that doesn't support this change.